src/backend/executor/functions.c

```diff
@@ -971,11 +971,21 @@ postquel_sub_params(SQLFunctionCachePtr fcache,
 			 * because of possible function inlining during planning.)
 			 */
 			prm->isnull = fcinfo->args[i].isnull;
-			prm->value = MakeExpandedObjectReadOnly(fcinfo->args[i].value,
+			if (i >= fcache->pinfo->nargs)
+			{
+				prm->value = MakeExpandedObjectReadOnly(fcinfo->args[i].value,
+													prm->isnull,
+													get_typlen(0));
+				prm->ptype = 0;
```

Comment:
This ptype being zero doesn't seem right to me. Btw, this issue seem to be community PG issue, we should report it in the community and try to find the solution collectively.

Reply:
agree with @Deepesh125 , @Suthapalli-Ramya-satya-vasavi-srija  is there a way we can fix this on the extension? If not, then we may need to report it in the community.

@Deepesh125 more info in the Jira.

Reply:
if this is an issue in engine code, we should not fix it in extension.